public/assets/sass/pages/resources/important-links.scss

```
@@ -0,0 +1,56 @@
+@import "../../util/_sass-mixins";
+@import "../../util/variables";
+
+section {
+    max-width: 600px;
+    margin: auto;
+    text-align: center;
+
+    .list {
+        list-style-type: none;
+        padding-left: 0;
+
+        .list-item-content {
+            border: 2px solid var(--mathsoc-pink-2);
+            border-radius: 3em;
+            margin: 1em;
+
+            .list-item-link {
+                text-decoration: none;
+                display: flex;
+                align-items: center;
+                justify-content: space-between;
+                padding: 1em;
+
+                .list-item-icon {
+                    height: 100%;
+                    width: auto;
+                    padding-right: 1em;
+                }
+                
+                .list-item-title {
+                    margin: 0;
+                    font-weight: 400;
+                    color: var(--mathsoc-pink-2);
+                    padding-left: 1em;
+                }
+            }
+        }
```

```suggestion
            }
            
        &:hover, 
        &:focus-visible {
          background-color: var(--mathsoc-pink-2);
          color: white;
          
          .list-item-title {
            color: white;
          }

          .list-item-icon {
            filter: invert(100%);
          }          
        }
        }
```

Since we're using SCSS, we can define the `hover` styles like this (using `&:hover` in the main block), keeping them in the main block with the rest of the element's style and reducing duplication! the approach right now works, but something like this is generally cleaner!